integration/container/mounts_linux_test.go

```diff
@@ -265,3 +267,36 @@ func TestContainerBindMountNonRecursive(t *testing.T) {
 		poll.WaitOn(t, container.IsSuccessful(ctx, client, c), poll.WithDelay(100*time.Millisecond))
 	}
 }
+
+func TestMountSubPath(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon)
```

Comment:
This should work on a remote daemon as well?

Reply:
Thank you for the review. Yes. This was because I was starting a daemon directly, which was not needed. I will update the test to only use a client from the test environment.